docs/table/access_table.rst

```diff
@@ -632,8 +651,8 @@ column name header::
   3 .. 40
   5 .. 60
 
-In order to see all the data values for a multidimensional column use the
-column representation.  This uses the standard `numpy` mechanism for printing
+In order to see all of the data values for a multidimensional column use the
+column representation. This uses the standard `NumPy` mechanism for printing
```

Comment:
This is about `numpy` code (specifically the `repr` for arrays) not the project.